### PR TITLE
Added the collapsible by years now.

### PR DESCRIPTION
**Pull request by MichaelGee**
Closes #31

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -437,6 +466,7 @@ <h2>#TwitterThrowback</h2>
                     }
                     return all;
                 }, {});
+                console.log(results);
```
Comment (oneEyedSunday) [inline docs/index.html]: Could you remove console.log from the PR? @MichaelGee

File: docs/index.html [hunk 2]
```diff
@@ -95,6 +95,25 @@
             min-width: 120px;
         }
 
+        .year_button {
+            background-color:#0d47a1;
+            display:block !important;
+            width: 50%;
```
Comment (oneEyedSunday) [inline docs/index.html:101]: Style wise for a PC, could we just make the minimizer a small icon like [this](https://feathericons.com/?query=minimize). Its a style choice though.

File: docs/index.html (hunk 1 above)
Comment (MichaelGee) [reply docs/index.html]: Yeah. Sorry about that. I missed that part

File: docs/index.html (hunk 2 above)
Comment (MichaelGee) [reply docs/index.html:101]: I dont...understand. I thought we wanted a button that shows the year?

**Comment by oneEyedSunday** [comment]
Yh. Like I said. This is just a personal style choice. Feel free to
disregard. Just something minimalist. The colours are getting rather
obvious on the screen. The blue against the background isn't so subtle

**Comment by shalvah** [comment]
This looks interesting. For a UI change like this, we need more screenshots.

- How does it look expanded?
- How does on mobile (expanded and collapsed)?

Also, when results are loaded, which is the default view (collapsed or expanded)?

Don't forget to remove that `snapshots` folder.

@oneEyedSunday, this LGTM, but then, my eyes aren't the best.😅 If you think it can look better, then it's best to wait till this is merged and then send in that as an improvement.

**Comment by shalvah** [comment]
Btw, @MichaelGee, before/after screenshots means "how did the frontend look before you added this change vs after?" Not before/after entering a username. This means both screenshots should be of the page in "results" state.

**Comment by oneEyedSunday** [comment]
Hahaha. I've been looking at bootstrap blue for a long time. I'd do just
that if I get to it. This issue had been on my plate for a while.

**Comment by MichaelGee** [comment]
So i should send more snapshots right now?

**Comment by MichaelGee** [comment]
screencapture-localhost-5500-2019-08-19-12_24_06
screencapture-localhost-5500-2019-08-19-12_24_53
screencapture-localhost-5500-2019-08-19-12_26_56
screencapture-localhost-5500-2019-08-19-12_27_12

**Comment by shalvah** [comment]
Yes, more screenshots. And don't forget to remove the ones you added to the code.

File: docs/index.html [hunk 3]
```diff
@@ -466,6 +491,13 @@ <h2>#TwitterThrowback</h2>
         }, 1000);
     }
 
+    //Collapsible Effect
+    function minimizer(event){
```
Comment (shalvah) [inline docs/index.html]: It's a function that does something. Please name it with a verb (like `minimize`). But I'd suggest something more accurate and descriptive, like `toggleTweetsView` or even `showOrHideTweetsInYear`. Long, but descriptive.

```suggestion
    function showOrHideTweetsInYear(event){
```

**Comment by MichaelGee** [comment]
Ok. Done.

**Review by shalvah — CHANGES_REQUESTED** [review]
Nice work.💪

File: docs/index.html (hunk 3 above)
Comment (MichaelGee) [reply docs/index.html]: Ok. Sure.

**Comment by shalvah** [comment]
Nice! Last questions:
- I'm guessing expanded is default? Can we make it collapsed by default?
- Does the cursor change on hover so it's obvious the years are clickable?

**Comment by MichaelGee** [comment]
Expand isnt default. Once the page loads, its closed, so all u have to do is click on the button to reveal the Tweet for each year.

File: docs/index.html [hunk 4]
```diff
@@ -388,9 +409,12 @@ <h2>#TwitterThrowback</h2>
                     if (tweets[year].length) {
                         hasContent = true;
                         tweetsHtml.push(`<div>`);
-                        tweetsHtml.push(`<h3>${year}</h3>`);
+                        tweetsHtml.push(`<div class="year_button" onclick="minimizer(event)">${year}</div>`);
```
Comment (shalvah) [inline docs/index.html:412]: Forgot to update this?

```suggestion
                        tweetsHtml.push(`<div class="year_button" onclick="showOrHideTweetsInYear(event)">${year}</div>`);

```

**Comment by shalvah** [comment]
> Does the cursor change on hover so it's obvious the years are clickable?

You didn't answer this?

**Pull request merged**